src/containers/guest-home-page/styles/WelcomeBlockStyles.js

```diff
@@ -0,0 +1,72 @@
+import theme from '~/styles/app-theme/custom-mui.styles'
+import desktopBg from '~/img/guest-home-page/desktopBg.svg'
```

Comment:
All images should be moved to assets/images folder.

Reply:
Fixed.